src/backend/src/prisma/migrations/20230604173138_link_to_model/migration.sql

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
is the plan to just delete all the existing links?

Reply:
Negative i have to add the insertions manually once i finish everything else